Aerolyzer/app/forms.py

```diff
@@ -1,23 +1,23 @@
-#files.py
 import re
 from django import forms
 from django.contrib.auth.models import User
 from django.utils.translation import ugettext_lazy as _
- 
+
 class RegistrationForm(forms.Form):
+    username = forms.RegexField(regex=r'^\w+$', widget=forms.TextInput(attrs=dict(required=True, max_length=30)), label=_("Username"), error_messages={ 'invalid': _("Must contain only letters, numbers and underscores. Try again!") })

```

Comment:
split line. Same for the others. Line length shouldn't exceed 100 characters (as per our Python linting standards)